packages/runner/cypress/e2e/reporter.errors.cy.js

```diff
@@ -1,782 +0,0 @@
-import { verify, verifyInternalFailure } from '../support/verify-failures'
-
-describe('errors ui', () => {
-  describe('assertion failures', () => {
-    const file = 'assertions_spec.js'
-
-    verify.it('with expect().<foo>', {
-      file,
-      column: 25,
-      message: `expected 'actual' to equal 'expected'`,
-    })
-
-    verify.it('with assert()', {
-      file,
-      column: '(5|12)', // (chrome|firefox)
-      message: `should be true`,
-    })
-
-    verify.it('with assert.<foo>()', {
-      file,
-      column: 12,
-      message: `expected 'actual' to equal 'expected'`,
-    })
-  })
-
-  describe('exception failures', () => {
-    const file = 'exceptions_spec.js'
-
-    verify.it('in spec file', {
-      file,
-      column: 10,
-      message: 'bar is not a function',
-    })
-
-    verify.it('in file outside project', {
-      file,
-      message: 'An outside error',
-      regex: /todos\/throws\-error\.js:5:9/,
-      codeFrameText: `thrownewError('An outside error')`,
-      verifyOpenInIde: false,
-    })
-  })
-
-  describe('hooks', { viewportHeight: 900 }, () => {
-    const file = 'hooks_spec.js'
-
-    // https://github.com/cypress-io/cypress/issues/8214
-    // https://github.com/cypress-io/cypress/issues/8288
-    // https://github.com/cypress-io/cypress/issues/8350
-    verify.it('errors when a hook is nested in another hook', {
-      file,
-      column: '(7|18)', // (chrome|firefox)
-      codeFrameText: 'beforeEach(()=>',
-      message: `Cypress detected you registered a(n) beforeEach hook while a test was running`,
-    })
-  })
-
-  describe('commands', () => {
-    const file = 'commands_spec.js'
-
-    verify.it('failure', {
-      file,
-      column: 8,
-      message: 'Timed out retrying after 0ms: Expected to find element: #does-not-exist, but never found it',
-    })
-
-    verify.it('chained failure', {
-      file,
-      column: 20,
-      message: 'Timed out retrying after 0ms: Expected to find element: #does-not-exist, but never found it',
-    })
-  })
-
-  describe('cy.then', () => {
```

Comment:
Did his guy might have been dropped on accident? I'm on seeing these cy.then tests in app/reporter.errors.cy

Reply:
Good catch! I'll add those back.